codemon/CodemonHelp.py

```diff
@@ -1,11 +1,29 @@
 from clint.textui import colored
+import os
 
 def showHelp():
-  print("           ---CODEMON---          ")
-  print(colored.cyan("  a CLI tool for competitive coders"))
-  print("\nCOMMANDS: \n")
-  print("codemon - - - - - - - - - - - - - - - shows help")
-  print("codemon init <contestName>  - - - - - initialises a contest")
-  print("codemon init -n <file>  - - - - - - - creates file with given name")
-  print("codemon listen  - - - - - - - - - - - compiles and gives output")
-  print("codemon practice <dirName>  - - - - - starts a practice session")
+  os.system('cls sys')

```

Comment:
We don't want to clear the screen every time user looks for help, sometimes we have useful data which we want to keep on the terminal. Also the command will show error on UNIX/Linux.